atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -344,46 +290,23 @@ const CreateCrashRecord = ({ client }) => {
           </FormGroup>
           <FormGroup row>
             <Col md="3">
-              <Label htmlFor="date-input">Crash Time</Label>
-            </Col>
-            <Col xs="12" md="9">
-              <Input
-                type="time"
-                id="time-input"
-                name="time-input"
-                placeholder="time"
-                value={crashTime}
-                invalid={isFieldInvalid(crashTime)}
-                onChange={e => setCrashTime(e.target.value)}
-              />
-              {isFieldInvalid(crashTime) ? (
-                <FormFeedback>Crash Time can't be blank.</FormFeedback>
-              ) : (
-                <FormText className="help-block">
-                  Please enter the Crash Time
-                </FormText>
-              )}
-            </Col>
-          </FormGroup>
-          <FormGroup row>
-            <Col md="3">
-              <Label htmlFor="date-input">Crash Date</Label>
+              <Label htmlFor="date-input">Crash Timestamp</Label>
             </Col>
             <Col xs="12" md="9">
               <Input
-                type="date"
-                id="date-input"
-                name="date-input"
-                placeholder="date"
-                value={crashDate}
-                invalid={isFieldInvalid(crashDate)}
-                onChange={e => setCrashDate(e.target.value)}
+                type="datetime-local"
+                id="timestamp-input"
+                name="timestamp-input"
+                placeholder="timestamp"
+                value={crashTimestamp}
+                invalid={isFieldInvalid(crashTimestamp)}
+                onChange={e => setCrashTimestamp(e.target.value)}
               />
-              {isFieldInvalid(crashDate) ? (
-                <FormFeedback>Crash Date can't be blank.</FormFeedback>
+              {isFieldInvalid(crashTimestamp) ? (
+                <FormFeedback>Crash Timestamp can't be blank.</FormFeedback>
```

Comment:
this feedback never appears because we are using a default value for `crashTimestamp`.

weirdly, i don't see that default value in the data input when i load the form. possibly because it's not formatted correctly as a string (e.g. `2024-07-18T13:05`) ?

Reply:
ah yeah now i'm seeing the warning message:
```
react-dom.development.js:1734 The specified value "2024-07-15" does not conform to the required format.  The format is "yyyy-MM-ddThh:mm" followed by optional ":ss" or ":ss.SSS".
```

Reply:
fixed dis